web/content/docs/benchmarks/reactive-transport/kineticreactant_allascomponents/KineticReactant2.pandoc

```diff
@@ -0,0 +1,92 @@
++++
+author = "Johannes Boog"
+weight = 143
+project = "Parabolic/ReactiveTransport/KineticReactant_AllAsComponents/"
+date = "2010-06-25T14:41:09+01:00"
+title = "Solute transport including kinetic reaction"
+
+[menu]
+  [menu.benchmarks]
+    parent = "Reactive transport"
+
++++
+
+{{< data-link >}}
+
+
+
+
+## Overview
+
+This scenario describes the transport of two solutes (Snythetica and Syntheticb) through a saturated media.
+Both solutes react to Productc according to $\text{Productd}=\text{Synthetica}+0.5~\text{Syntheticb}$.
```

Comment:
Product**c** in the formula, I guess.

Reply:
No, its Product**d** .

Reply:
There is a mismatch between the sentence and the formula. If it's 'd', then:
```suggestion
Both solutes react to Productd according to $\text{Productd}=\text{Synthetica}+0.5~\text{Syntheticb}$.
```
I may miss something, though. If it's 'c' and then 'd', and you are sure about it, then it's fine, just checking.

Reply:
OMG, I didn't see that. 

Now updated.